package.json

```diff
@@ -16,6 +17,7 @@
   "dependencies": {
     "bcrypt-nodejs": "0.0.3",
     "body-parser": "^1.18.3",
+    "dotenv": "^8.2.0",
```

Comment:
remove the carret from the package versions, it can lead to nasty things :)

Reply:
Yes. Forgot about those. Thank you